ibis/pandas/execution/temporal.py

```diff
@@ -33,15 +33,20 @@ def execute_extract_timestamp_field_timestamp(op, data, **kwargs):
     return getattr(data, field_name)
 
 
+@execute_node.register(ops.ExtractTemporalField, pd.Series)
```

Comment:
`execute_extract_timestamp_field_series` was moved up to be close to above function `execute_extract_timestamp_field_timestamp`